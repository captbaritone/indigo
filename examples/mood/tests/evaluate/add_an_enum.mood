enum Foo {
    Bar
}
pub fn test(): f64 {
    Foo::Bar + 1.0_f64
}