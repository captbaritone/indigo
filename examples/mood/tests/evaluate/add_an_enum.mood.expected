-----------------
INPUT
----------------- 
enum Foo {
    Bar
}
pub fn test(): f64 {
    Foo::Bar + 1.0_f64
}
-----------------
OUTPUT
-----------------
Error: Expected a number.:
 --> add_an_enum.mood:5:5

  | pub fn test(): f64 {
5 |     Foo::Bar + 1.0_f64
  |     ^^^^^^^^ This expression is not numeric.
  | }
